docs/semgrep-cloud-platform/dashboard.md

```diff
@@ -25,31 +25,31 @@ Object.entries(frontMatter).filter(
+The **Code** widget displays high-level security analytics across your entire organization. This includes:
 <dl>
-	<dt>Open findings</dt>
-	<dd>The number of open findings over the **given time period** next to the calendar icon. The badge number indicates whether this number has gone up or down compared to the previous timeframe. The number badge compares the most recent number of open findings against the previous timeframe.</dd>
 	<dt>High severity</dt>
-	<dd><a href='/semgrep-code/findings/#understanding-the-findings-data'>High severity findings</a> are findings generated by a rule with the severity value set to <code>ERROR</code>. These include security backdoors, highly vulnerable code, and so on. The number badge compares the number of high severity findings within the given time period against the previous time period.</dd>
-	<dt>Pull request (PR) fix rate</dt>
-	<dd>The percentage of findings that were fixed when findings were surfaced to developers via PR comments in previous scans. The number badge compares PR fix rate in the given time period against the previous time period.</dd>
+	<dd>Findings generated by a rule with the severity value set to <code>ERROR</code>. These include security backdoors, and highly vulnerable code. If you filter for another time period than <b>All time</b> the displayed number badge compares the number of high-severity findings within the given time period against the previous time period.</dd>
+	<dt>Open findings</dt>
+	<dd>The number of open findings over the <b>given time period</b>. The badge number indicates whether this number has gone up or down compared to the previous timeframe. If you filter for another time period than <b>All time</b> the displayed number badge compares the most recent number of open findings against the previous timeframe.</dd>
+	<dt>Comment fix rate</dt>
+	<dd>The percentage of findings that were fixed when findings surfaced to developers through PR or MR comments in previous scans. If you filter for another time period than <b>All time</b> the displayed number badge compares PR and MR fix rate in the given time period against the previous time period.</dd>
 </dl>
 
 ## Filtering findings by time
 
-The Dashboard displays data from scans for the **last one month** by default. This time range can be set to a broader or narrower value. By broadening the time range, security teams are able to see total numbers and statistics across an entire time period. Narrow time ranges can give insights into the most recent vulnerabilities creeping into the project.
+The Dashboard displays data from scans for the **last 1 month** by default. This time range can be set to a broader or narrower value. By broadening the time range, security teams are able to see total numbers and statistics across an entire time period. Narrow time ranges can give insights into the most recent vulnerabilities creeping into the project.
```

Comment:
Not sure if it is still one month. I checked and it's set to "All time".

Reply:
@s-santillan I was not sure if I see All time because I selected it once...

```suggestion
The Dashboard displays data from scans for the **All time** by default. This time range can be set to a narrower value. By broadening the time range, security teams are able to see total numbers and statistics across an entire time period. Narrow time ranges can give insights into the most recent vulnerabilities creeping into the project.
```